app/models/query_command.rb

```diff
@@ -136,7 +136,7 @@ def query_limit
   end
 
   def save_results(ppd, query, options)
-    Rails.logger.debug { "About to ask DsAPI query: #{query.to_json}" }
+    Rails.logger.debug { "Current DsAPI query: #{query.to_json}" } if Rails.env.development?
```

Comment:
Maybe this should be changed to SAPI-NT once we make the switch, otherwise changes look good 👍